server/server.go

```diff
@@ -153,6 +153,13 @@ type Config struct {
 
 	// This limiter is called for every started and finished gRPC request.
 	GrpcMethodLimiter GrpcInflightMethodLimiter `yaml:"-"`
+
+	Throughput Throughput `yaml:"throughput"`
```

Comment:
i also prefer nested config, but i noticed that most of the other configs in server.Config are flattened (LogLevel, LogFormat instead of Log.Level, Log.Format). I think it's better to stay consistent because this is a shared library

so can you just move the two fields and prefix them with `Throughput`?